### PR TITLE
Added IC_7459, IC_74138. IC_7445_new and IC_74138_new using combinational.py 

### PR DESCRIPTION
**Pull request by shrutig**
Added IC 74138 for decoder  and IC 7459 for 2-input and 3-input AND-OR inverter gate

### CODE REVIEW

File: BinPy/ic/series_7400.py [hunk 1]
```diff
@@ -2242,3 +2242,83 @@ def run(self):
...
+        output[8] = NOR(temp[2],temp[3]).output()
+        
+        if self.pins[7] == 0 and self.pins[14] == 1:
+            for i in self.outputConnector:
+                self.outputConnector[i].state = output[i]
+            return output
+        else:
+            print("Ground and VCC pins have not been configured correctly.")
+
+class IC_74138(Base_16pin):
```
Comment (rajat974) [inline BinPy/ic/series_7400.py]: put this in 16 pins section i.e. below all 14 pins ics

**Comment by shrutig** [comment]
The changed file now has IC_7445_new and IC_74138_new using the DEMUX functionality from combinational.py and the IC_74138 has also been corrected

**Comment by rajat974** [comment]
@mohitgujarathi14 The functionality of directly instance to conversion of output in mux is implemented. So there was no need, but it is fine .

**Comment by jayrambhia** [comment]
You need to add tests and examples for the same.

**Comment by debugger22** [comment]
@rvraghav93 merge conflict.

**Comment by ghost** [comment]
#209

**Pull request closed without merging**